Update URLs from kuzudb.com to kuzudb.github.io (#647)
kuzudb/kuzu-docs: src/content/docs/extensions/index.mdx

### Host your own extension server

> By default, Kuzu installs official extensions from `https://extension.kuzudb.github.io/`. However, if you are running Kuzu in an environment without internet access (e.g., an air-gapped environment), you can host your own extension server using Docker.

Author: Since the extension repo isn't up any more maybe this should be updated to
```suggestion
You can host your own extension server for Kuzu using Docker.
```
